src/html-conventions-lib/connectors-with-same-name.xsl

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns:math="http://www.w3.org/2005/xpath-functions/math"
+    xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl" xmlns:fn="http://www.w3.org/2005/xpath-functions"
+    exclude-result-prefixes="xs math xd xsl uml xmi umldi dc fn"
+    xmlns:uml="http://www.omg.org/spec/UML/20131001"
+    xmlns:xmi="http://www.omg.org/spec/XMI/20131001"
+    xmlns:umldi="http://www.omg.org/spec/UML/20131001/UMLDI"
+    xmlns:dc="http://www.omg.org/spec/UML/20131001/UMLDC" xmlns:owl="http://www.w3.org/2002/07/owl#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dct="http://purl.org/dc/terms/"
+    xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+    xmlns:f="http://https://github.com/costezki/model2owl#" version="3.0">
+
+    <xsl:import href="../common/checkers.xsl"/>
+    <xsl:import href="utils-html-conventions.xsl"/>
+
+    <xd:doc>
+        <xd:desc>Applying the checkers to a group of connectors with same name</xd:desc>
+    </xd:doc>
+
+    <xsl:template name="connectorsWithSameName">
+        <xsl:variable name="root" select="root()"/>
+        <xsl:variable name="distinctNames" select="f:getDistinctConnectorsNames($root)"/>
+        <h1>Connectors with the same name</h1>
```

Comment:
`<h1>Properties with multiple usages</h1>`
Move this section down, after the classes/enumerations/datatypes, just before any connector section.